diff --git a/lua/lvim/core/project.lua b/lua/lvim/core/project.lua
index 17473c55..47c57904 100644
--- a/lua/lvim/core/project.lua
+++ b/lua/lvim/core/project.lua
@@ -23,7 +23,22 @@ function M.config()
 
     -- All the patterns used to detect root dir, when **"pattern"** is in
     -- detection_methods
-    patterns = { ".git", "_darcs", ".hg", ".bzr", ".svn", "Makefile", "package.json", "pom.xml" },
+    patterns = {
+      ".git",
+      "_darcs",
+      ".hg",
+      ".bzr",
+      ".svn",
+      "Makefile",
+      "package.json",
+      "pom.xml",
+      ".root",
+      ".project",
+      ".workspace",
+      "WORKSPACE",
+      "Cargo.toml",
+      "compile_commands.json",
+    },
 
     -- Table of lsp clients to ignore by name
     -- eg: { "efm", ... }
