diff --git a/lua/lvim/core/terminal.lua b/lua/lvim/core/terminal.lua
index 6e0cb582..887e54fa 100644
--- a/lua/lvim/core/terminal.lua
+++ b/lua/lvim/core/terminal.lua
@@ -7,7 +7,8 @@ M.config = function()
     on_config_done = nil,
     -- size can be a number or function which is passed the current terminal
     size = 20,
-    open_mapping = [[<c-\>]],
+    -- open_mapping = [[<c-\>]],
+    open_mapping = nil,
     hide_numbers = true, -- hide the number column in toggleterm buffers
     shade_filetypes = {},
     shade_terminals = true,
@@ -41,8 +42,8 @@ M.config = function()
     -- lvim.builtin.terminal.execs[#lvim.builtin.terminal.execs+1] = {"gdb", "tg", "GNU Debugger"}
     -- TODO: pls add mappings in which key and refactor this
     execs = {
-      { nil, "<M-1>", "Horizontal Terminal", "horizontal", 0.3 },
-      { nil, "<M-2>", "Vertical Terminal", "vertical", 0.4 },
+      { nil, "<M-1>", "Horizontal Terminal", "horizontal", 0.38 },
+      { nil, "<M-2>", "Vertical Terminal", "vertical", 0.38 },
       { nil, "<M-3>", "Float Terminal", "float", nil },
     },
   }
