diff --git a/bin/dotbot b/bin/dotbot
index 6232a50..c7c41c7 100755
--- a/bin/dotbot
+++ b/bin/dotbot
@@ -7,7 +7,7 @@
 # is useful because we don't know the name of the python binary.
 
 ''':' # begin python string; this line is interpreted by the shell as `:`
-command -v python  >/dev/null 2>&1 && exec python  "$0" "$@"
+# command -v python  >/dev/null 2>&1 && exec python  "$0" "$@"
 command -v python3 >/dev/null 2>&1 && exec python3 "$0" "$@"
 command -v python2 >/dev/null 2>&1 && exec python2 "$0" "$@"
 >&2 echo "error: cannot find python"
